compiler/src/dotty/tools/repl/Rendering.scala

```diff
@@ -21,24 +21,24 @@ import dotc.core.StdNames.str
  *       `Rendering` is no longer valid.
  */
 private[repl] class Rendering(compiler: ReplCompiler,
-                              private var currentClassLoader: Option[ClassLoader] = None) {
+                              parentClassLoader: Option[ClassLoader] = None) {
 
-  private def classLoader()(implicit ctx: Context) =
-    currentClassLoader.getOrElse {
-      import java.net.{URL, URLClassLoader}
-
-      /** the compiler's classpath, as URL's */
-      val compilerClasspath: Seq[URL] = ctx.platform.classPath(ctx).asURLs
+  private var _classLoader: ClassLoader = _
```

Comment:
`private[this]`. Also we usuallly call these variables `myFoo` instead of  `_foo`.